libs/libc/machine/arm/armv7-a/arch_elf.c

```diff
@@ -163,7 +172,12 @@ int up_relocate(FAR const Elf32_Rel *rel, FAR const Elf32_Sym *sym,
           }
 
         offset += sym->st_value - addr;
+
+#ifdef CONFIG_ARM_THUMB
+        if (offset & 2 || offset < (int32_t) 0xfe000000 ||
```

Comment:
```suggestion
        if ((offset & 2) != 0 || offset < (int32_t) 0xfe000000 ||
```

Reply:
@pkarashchenko 

Done.
I also applied the same coding style to existing code.